Makefile

```diff
@@ -182,9 +182,15 @@ ifndef PCRE_LDFLAGS
  endif
 endif
 
+
+migrations:
+	$(ENV_SCRIPT) nim c -r $(NIM_PARAMS) \
+		nim_status/lib/migrations/sql_generate.nim > nim_status/lib/migrations/sql_scripts.nim
+
+
 NIMSTATUS ?= build/nim_status.a
 
-$(NIMSTATUS): $(SQLCIPHER)
+$(NIMSTATUS): $(SQLCIPHER) migrations
```

Comment:
In places where you have the `migrations` phony target as a dependency, instead use the `$(MIGRATIONS)` target (see my other comments).